Library/Aliases/git-hub.rb

```diff
@@ -0,0 +1 @@
+Library/Formula/git-cmd-hub.rb
```

Comment:
Also need to remove the `.rb` extension. Link should be to `../Formula/git-cmd-hub.rb`